packages/site/src/components/App.js

```diff
@@ -60,33 +61,59 @@ const Root = createStyledComponent('div', styles.app, {
 const Nav = createStyledComponent(_Nav, styles.nav);
 const Main = createStyledComponent('main', styles.main);
 
-const getDefaultDemo = (demos: Object): string => Object.keys(demos)[0];
-
 export default function App({ className, demos }: Props) {
+  const routes = [];
```

Comment:
Routes configuration is starting to get a bit unwieldy here.  We'll should consider breaking this out into a separate `routes` component.

Reply:
Good call. Done.